build_image.sh

```diff
@@ -133,6 +133,7 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     sed -i -e "s/%%IMAGE_VERSION%%/$IMAGE_VERSION/g" files/Aboot/boot0
     pushd files/Aboot && zip -g $OLDPWD/$OUTPUT_ABOOT_IMAGE boot0; popd
     pushd files/Aboot && zip -g $OLDPWD/$ABOOT_BOOT_IMAGE boot0; popd
+    pushd files/image_config/secureboot && zip -g $OLDPWD/$OUTPUT_ABOOT_IMAGE whitelist; popd
```

Comment:
>whitelist [](start = 78, length = 9)

Name it as `whitelist_paths.txt` ?

Reply:
Can we use whitelist_paths? Do we need to remove the .txt when added into the image package?

Reply:
Changed to whitelist_paths.conf